hydrus/resources.py

```diff
@@ -494,14 +490,14 @@ def get(self, category: str) -> Response:
         # Check for collection
         if category in get_doc().collections:
             # type: Union[Dict[str,Any],Dict[int,str]]
-            response = {
+            response={
                 "@context": get_doc().collections[category]["context"].generate()}
             return set_response_headers(jsonify(response))
         # Check for non collection class
         elif category in get_doc().parsed_classes:
-            response = {
+            response={
                 "@context": get_doc().parsed_classes[category]["context"].generate()}
             return set_response_headers(jsonify(response))
         else:
-            response = {404: "NOT FOUND"}
+            response={404: "NOT FOUND"}
```

Comment:
Do not remove spaces surrounding operators, binary operators should have a single space on either side to facilitate readability.

Reply:
Oh, ok. I ran a pep8 checker which was the reason for the change.